deployment/src/main/java/org/wso2/carbon/testgrid/deployment/DeployerServiceImpl.java

```diff
@@ -127,11 +102,9 @@ private String getKubernetesMaster(String location){
         try {
             InputStream inputStream = new FileInputStream(location);
             prop.load(inputStream);
-
-            System.out.println(inputStream);
-
         } catch (IOException e) {
-            e.printStackTrace();
+            String msg = "Error getting KUBERNETES_MASTER environment variable";
+            log.error(msg, e);
```

Comment:
Better to throw an exception rather than logging it here.

Reply:
Fixed in commit 21bb7f6